utils/motorcontroller/defaultconfig/NeoDefaults.java

```diff
@@ -0,0 +1,27 @@
+package frc.team670.mustanglib.utils.motorcontroller.defaultconfig;
+
+import frc.team670.mustanglib.utils.motorcontroller.MotorConfig.SparkMAXFactory.Config;
+
+public class NeoDefaults {
```

Comment:
These aren't really "NeoDefaults" - they're "SparkMaxDefaults". I believe. I haven't looked at the CAN frame stuff in a while but my memory is that they don't change depending on what kind of motor the SparkMax is controlling (note that it can control brushed motors as well such as CIMs or 775 Pros).